nixio/test/test_multi_tag.py

```diff
@@ -402,8 +402,10 @@ def test_multi_tag_feature_data(self):
         assert(feat_data.size == 10)
         assert(np.sum(feat_data) == 55)
 
-        data_view = self.feature_tag.retrieve_feature_data(9, 0)
-        assert(np.sum(data_view[:, :]) == 9055)
+        # disabled, don't understand how it could ever have worked,
+        # there are only 3 positions
+        # data_view = self.feature_tag.retrieve_feature_data(9, 0)
```

Comment:
huh... hm. Might be good to know why and if there's a bug we need to look at.

Reply:
Might be related to that thing I found recently and resulted in #337. Seems like it's a different issue, but they both have to do with indexing outside size limits.